sockeye/utils.py

```diff
@@ -791,7 +791,7 @@ def write_metrics_file(metrics: List[Dict[str, Any]], path: str):
     """
     with open(path, 'w') as metrics_out:
         for checkpoint, metric_dict in enumerate(metrics, 1):
-            metrics_str = "\t".join(["{}={}".format(name, value) for name, value in sorted(metric_dict.items())])
+            metrics_str = "\t".join(["{}={:f}".format(name, value) for name, value in sorted(metric_dict.items())])
```

Comment:
Why the change to fixed point?

Reply:
This is for the read_metrics() @[L780](https://github.com/awslabs/sockeye/blob/09068cdeba9be538fccd78496a6b60a26d2a995f/sockeye/utils.py#L780) from utils.py that assumes everything is float.

Reply:
Now doing the check in the read_metrics().